index.html

```diff
@@ -202,5 +202,57 @@ <h3 class="br-header">Multi-Post Stories</h3>
       </div>
     </div>
   </section>
+  <!-- Contact Section -->
+  <section id="contact-section" class="center-align">
+    <div class="container">
+      <div class="contact-form">
+        <h2 class="recent-works robo-head center-align">Contact us</h2>
+        <p class="text">
+          If you have an application you are interested in developing, a feature 
+          that you need built or a project that needs coding. 
+          I'd love to help with it.
+        </p>
```

Comment:
- [ ] Kindly resize the font to be similar to the Figma template. We are using 1.25 rem = 20 px and Figma template uses 16 px = 1 rem. 🧾